src/containers/course-sections-list/CourseSectionsList.tsx

```diff
@@ -99,9 +105,32 @@ const CourseSectionsList: FC<CourseSectionsListProps> = ({
     addNewSection?.(currentSectionIndex)
   }
 
+  const openAddCourseTemplateModal = () => {
+    closeMenu()
+    setIsAddedClicked(false)
+    openModal({
+      component: <AddCourseTemplateModal closeModal={closeModal} />
+    })
+  }
+
+  const addActivityMenuItems = [
```

Comment:
why you decide to pass `menu items` here ? 
Maybe better to import it from `CourseSectionsList.constants.tsx` ?

Reply:
I did this because in the addActivityMenuItems object I use functions that depend on the props that the CourseSectionsList component accepts.
Also in that object I call functions that change states. 
Therefore I don't see the point in destructuring the comoponet